plugins/permission-node/report.api.md

```diff
@@ -37,20 +38,33 @@ export type ApplyConditionsRequest = {
 };
 
 // @public
-export type ApplyConditionsRequestEntry = IdentifiedPermissionMessage<{
-  resourceRef: string;
-  resourceType: string;
-  conditions: PermissionCriteria<PermissionCondition>;
-}>;
+export type ApplyConditionsRequestEntry = IdentifiedPermissionMessage<
+  | {
+      resourceRef: string;
+      resourceRefs?: undefined;
+      resourceType: string;
+      conditions: PermissionCriteria<PermissionCondition>;
+    }
+  | {
+      resourceRef?: undefined;
```

Comment:
would it just have been enough to make this a single `resourceRef: string | string[]`? Seems simpler at least to a casual observer. But maybe you wanted to deprecate the non-array one later down the line

Reply:
yeah that was my intention, but happy to change it to using only `resourceRef`

Reply:
I've pivoted to keeping just `resourceRef`